packages/components/package.json

```diff
@@ -66,6 +67,7 @@
   },
   "devDependencies": {
     "@types/react": "^18.2.46",
-    "@types/react-dom": "^18.2.18"
+    "@types/react-dom": "^18.2.18",
+    "style-dictionary": "^3.9.2"
```

Comment:
Each monorepo package needs the Style Dictionary package to be installed separately for the `yarn build` command.